lib/locales/urhobo/lorem/words.js

```diff
@@ -0,0 +1,9 @@
+module.exports = [
+  'Akpö ',
+  'Eki ',
+  'Isabato ',
+  'Ughegbe ',
+  'migwo',
+  'wadoo',
+];
+  
```

Comment:
You might want to lose the trailing spaces on this line, that's what's causing this red warning here.